include/s3select.h

```diff
@@ -1201,6 +1203,19 @@ void push_between_filter::builder(s3select* self, const char* a, const char* b)
 
   std::string between_function("#between#");
 
+  bool between{true};
+
+  for(size_t i=0;i<token.size();i++)
+  {
+    bsc::parse_info<> info = bsc::parse(token.c_str()+i, ( bsc::str_p("not") >> bsc::str_p("between")) , bsc::space_p);
```

Comment:
please validate that this part works as expected.
i.e. it accepts "not between" as it should